cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -176,6 +176,14 @@ func (mh *MessageHandle) KeepaliveCheckLoop(info *model.HubInfo, stopServe chan
 				return
 			}
 			klog.Infof("Node %s is still alive", info.NodeID)
+
+			// Reset is called after Stop or expired timer
+			if !keepaliveTicker.Stop() {
+				select {
+				case <-keepaliveTicker.C:
+				default:
+				}
+			}
```

Comment:
A convenient way to solve the conflict is drop all commits you made on your `pr-timer` branch. And then rebase master, then put the code on above in a single commit.

After this, force push `pr-timer` to your original repo.